java/choose-camera-controller/README.md

```diff
@@ -0,0 +1,33 @@
+# Choose camera controller
+
+Control the behaviour of the camera in a scene.
+
+![Choose camera controller](choose-camera-controller.png)
+
+## Use case
+
+The globe camera controller (the default camera controller in all new scenes) allows a user to explore the scene freely by zooming in/out and panning around the globe. The orbit camera controllers fix the camera to look at a target location or geoelement. A primary use case is for following moving objects like cars and planes.
+
+## How to use the sample
+
+The application loads with the default globe camera controller. To rotate and fix the scene around the aeroplane, exit globe mode by choosing the "Orbit camera around aeroplane" option (i.e. camera will now be fixed to the aeroplane). Choose the "Orbit camera around location" option to rotate and centre the scene around the location of the Upheaval Dome crater structure, or choose the "Free pan round the globe" option to return to default free navigation.
```

Comment:
I've just put in a request against the common design. We should probably change the spelling from Aeroplane -> Airplane (we're supposed to be targeting American English)